src/backlight/datasource/utils.py

```diff
@@ -65,6 +69,7 @@ def from_dataframe(
 
     mkt.symbol = symbol
     mkt.currency_unit = currency_unit
+    mkt.currency_entry = currency_entry
```

Comment:
Should be base_currency and quote_currency?

https://en.wikipedia.org/wiki/Currency_pair